python/src/nanoarrow/__init__.py

```diff
@@ -15,5 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from ._lib import Array, ArrayStream, ArrayView, Schema, c_version  # noqa: F401
-from .lib import array, array_stream, schema, array_view  # noqa: F401
+from nanoarrow._lib import cversion  # noqa: F401
+from nanoarrow.clib import (  # noqa: F401
+    cschema,
+    carray,
+    carray_stream,
+    cschema_view,
+    carray_view,
```

Comment:
nit: I wonder if it is considered more readable to use the naming convention `c_<object>` instead of `c<object>` since we are using underscores instead of camel case? This seems to be the pattern in the PyCapsule example[1] (see `c_api_import` variable name) and in the Arrow PyCapsule docs[2] (e.g. `__arrow_c_schema__`).

I'm also fine with it as-is if this is already a known pattern or is just the preferred option.

[1]https://docs.python.org/3/extending/extending.html#using-capsules
[2]https://arrow.apache.org/docs/dev/format/CDataInterface/PyCapsuleInterface.html#export-protocol